prime-router/src/main/kotlin/azure/LookupTableFunctions.kt

```diff
@@ -251,5 +258,10 @@ class LookupTableFunctions(
          * Name of the query parameter to show inactive tables.
          */
         const val showInactiveParamName = "showInactive"
+
+        /**
+         * Name of the query parameter to Sha256 tables checksum.
+         */
+        const val force = "force"
```

Comment:
Change the name of this val to something more descriptive.  forceHeaderParameter?

Reply:
Done.  Rename it to forceQueryParameter = "forceTableToCreate"